kernel/rros/factory.rs

```diff
@@ -306,7 +309,7 @@ impl device::Devnode for FactoryTypeDevnode {
         }
         kernelh::_kasprintf_2(
             bindings::GFP_KERNEL,
-            c_str!("evl/%s/%s").as_char_ptr(),
+            c_str!("rros/%s/%s").as_char_ptr(),
```

Comment:
Since you change it from `evl` to `rros`,  you also need to provide some methods for others to change the device path in linux-evl uapi headers.

Reply:
I checked some header files in the uapi, looking for parts related to `/dev/evl`, and found that only `control.h` directly specifies it. After I modified this part to `/dev/rros`, I was able to test my routines normally.

Reply:
No, other factories will also use this interface, like `/dev/evl/thread/clone`, and `/dev/evl/xbuf/clone`.  And I mean that you should not only change the interface here but also provide a way so that others can compile libevl directly. Otherwise, after someone compiles the libevl and runs the program, they may encounter problems such as "/dev/evl/thread/clone" not existing.

The easiest way is to add a note in `README.md` in [libevl](https://github.com/BUPT-OS/libevl.git) of our organization to notify users that they should change the uapi in advance.

Reply:
Ok, I know what you mean. I wonder if we can change the `lib/internal.c` in [libevl](https://github.com/BUPT-OS/libevl). Just change all `/dev/evl` to `/dev/rros`, then errors like `/dev/evl/**/** not exist` will not occur.

Reply:
> Ok, I know what you mean. I wonder if we can change the `lib/internal.c` in [libevl](https://github.com/BUPT-OS/libevl). Just change all `/dev/evl` to `/dev/rros`, then errors like `/dev/evl/**/** not exist` will not occur.

Sorry, I omit the `tests/mapfd.c` and `tests/helpers.c`, these files also need replacements.

Reply:
Yes. you can replace them.